### DISCUSSION TITLE
forge在pcl2中有官方下载源吗？

### DISCUSSION THREAD

**Discussion by hejiehao** [Meloong-Git/PCL#1533] [提问] [answered]
forge在pcl2中有官方下载源吗？

**Comment by WForst-Breeze** [comment, marked as answer]
从**具体使用过程看**应该不会从官方源下载，即使设置开启了 **尽量使用官方源** ，PCL2 **貌似**仍会使用镜像源下载 Forge（具体情况我不太清楚……从日志看至少是这样的
```
[Download] forge-1.19.4-45.0.40-installer.jar 754#：开始，起始点 0，https://download.mcbbs.net/maven/net/minecraftforge/forge/1.19.4-45.0.40/forge-1.19.4-45.0.40-installer.jar
```
如果您想从官方源下载 Forge 加载器，可以在 “手动安装包” 处打开官网手动安装
![image](https://user-images.githubusercontent.com/110760354/229128754-26ac200d-43d0-42c8-b513-8dded6777227.png)
